### PR TITLE
Add support for span open in IDE

### PR DESCRIPTION
**Pull request by purplexa**

### CODE REVIEW

File: shared/ui/index.tsx [hunk 1]
```diff
@@ -659,6 +664,130 @@ function listenForEvents(store) {
...
+										9999
+									),
+									atTop: true,
+								});*/
+							}
+						}
+
+						store.dispatch(
+							setCurrentTransactionSpan({
+								spanId,
```
Comment (bcanzanella) [inline shared/ui/index.tsx:773]: should "1" be removed here? probably don't want that for non-NR folks

File: shared/agent/src/providers/newrelic.ts [hunk 2]
```diff
@@ -3168,6 +3175,215 @@ export class NewRelicProvider
...
+			}
+			`,
+			{
+				accountId: request.accountId,
+			}
+		);
+
+		const partialData = response.actor.account.nrql.results.reduce(
+			(accum, val) => {
+				accum[val.endTimeSeconds * 1000] = accum[val.endTimeSeconds * 1000] || {};
```
Comment (eric-j) [inline shared/agent/src/providers/newrelic.ts:3287]: could we assign  `accum[val.endTimeSeconds * 1000]` to a variable to for less calculations and readability?

**Comment by bcanzanella** [comment]
one comment from me, otherwise looks great.

**Comment by bcanzanella** [comment]
just need a final review, but let's wait to merge it.

**Pull request merged**